minecraft_servers/melvin501/manifest.json

```diff
@@ -0,0 +1,74 @@
+{
+  "server_name": "melvin501",
+  "nice_name": "Melvin501",
+  "direct_ip": "play.melvin501.de",
+  "social": {
+    "web": "https://melvin501.de",
+    "youtube": "https://www.youtube.com/@melvin501de",
+    "discord": "https://melvin501.de/discord",
+    "tiktok": "melvin501.tiktok"
```

Comment:
custom urls are only supported for LabyMod partners at the moment.
Please use links by official services (e.g. discord.gg/xyz).

Documentation reference: https://github.com/LabyMod/server-media/blob/master/docs/Manifest.md#discord-url